standalone/src/models/commons/reference/types.ts

```diff
@@ -1,22 +1,6 @@
 import type { TBuilder } from '@/core';
 import { TCtpReference } from '@/graphql-types';
 
-// export interface BusinessUnitReference {
```

Comment:
Unrelated but I noticed this useless comment block and took the opportunity to remove it.